master_xml.py

```diff
@@ -308,6 +308,18 @@ def evaluateTokens(cards):
                         cards.get(tokenId)['released'] = True
                         card['related'].append(tokenId)
 
+                for template in ability.iter('TemplateId'):
+                    tokenId = template.attrib['V']
+                    token = cards.get(tokenId)
+                    if token != None and token.get('info') != None:
```

Comment:
Why the difference between line 294? Should 294 also check `token.get('info')`?

Reply:
Of the instances where `templateId` is used and 294 is called, there are no stub cards.

Only when I started looking for `TemplateId` did I start picking up some stub cards. I will add the logic to the other case because it'll protect the logic in the future, but it's worth noting that at the moment it's not actually necessary.

Reply:
Fixed.

I've applied the same logic to all the variations of token definitions. 

Again I've tested it vs the last data release and the only change is that Draugir is now correctly marked as released.